keras_nlp/models/roberta/roberta_preprocessor.py

```diff
@@ -133,24 +134,18 @@ class RobertaPreprocessor(Preprocessor):
     # Alternatively, you can create a preprocessor from your own vocabulary.
     # The usage is exactly the same as above.
     vocab = {
-        "<s>": 0,
-        "<pad>": 1,
-        "</s>": 2,
-        "Ġafter": 5,
-        "noon": 6,
-        "Ġsun": 7,
+        "<s>": 0, "<pad>": 1, "</s>": 2, "<mask>": 3, "Ġquick": 4, "Ġfox": 5,
     }
-    merges = ["Ġ a", "Ġ s", "Ġ n", "e r", "n o", "o n", "Ġs u", "Ġa f", "no on"]
-    merges += ["Ġsu n", "Ġaf t", "Ġaft er"]
-
+    merges = ["Ġ q", "u i", "c k", "ui ck", "Ġq uick", "Ġf o", "Ġfo x"]
     tokenizer = keras_nlp.models.RobertaTokenizer(
         vocabulary=vocab,
         merges=merges,
     )
-    preprocessor = keras_nlp.models.RobertaPreprocessor(
+    preprocessor = keras_nlp.models.RobertaMaskedLMPreprocessor(
         tokenizer=tokenizer,
         sequence_length=20,
     )
+    preprocessor(" quick fox quick fox")
```

Comment:
same: do we need the leading space?

Reply:
Yeah, this is probably confusing. I've been trying to come up with a version of "bring your own vocab" that is actually concise. Reworked this so no trailing space required.